spec/lib/telephony/notification_sender_spec.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
```

Comment:
since this is commented out let's remove it

Reply:
@svalexander ?? this is a ruby language directive, a comment with magic. Many ruby files in the repo have this.

Reply:
[reference here](https://docs.ruby-lang.org/en/3.1/syntax/comments_rdoc.html#label-Magic+Comments)

Reply:
It's not needed in specs generally, I think I'd prefer to remove it in this case.